deep_speech_2/data_utils/featurizer/audio_featurizer.py

```diff
@@ -24,26 +24,64 @@ class AudioFeaturizer(object):
                      corresponding to frequencies between [0, max_freq] are
                      returned.
     :types max_freq: None|float
+    :param target_sample_rate: Audio are resampled (if upsampling or
+                               downsampling is allowed) to this before
+                               extracting spectrogram features.
+    :type target_sample_rate: float
+    :param use_dB_normalization: Whether to normalize the audio to a certain
+                                 decibels before extracting the features.
```

Comment:
For comments, full name `decibels` is used for clarity, while in arguments a short name of `dB` is used instead.
I think it makes sense?